starfish/pipeline/filter/mean_high_pass.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+from functools import partial
+from numbers import Number
+from typing import Callable, Union, Tuple, Optional
+
+import numpy as np
+from skimage import img_as_uint
+
+from starfish.errors import DataFormatWarning
+from starfish.image import ImageStack
+from scipy.ndimage.filters import uniform_filter
+from ._base import FilterAlgorithmBase
+
+
+class MeanHighPass(FilterAlgorithmBase):
+
+    def __init__(
+            self, size: Number, is_volume: bool=False, verbose: bool=False, **kwargs
+    ) -> None:
+        """Mean high pass filter
+
+        Parameters
+        ----------
+        size : Number
+            width of the kernel
+        is_volume : bool
+            If True, 3d (z, y, x) volumes will be filtered, otherwise, filter 2d tiles independently.
+        verbose : bool
+            if True, report on filtering progress (default = False)
+
+        """
+
+        self.size = size
```

Comment:
It looks like `uniform_filter` supports a tuple of ints in the case that you want an anisotropic kernel. That might be nice to support here similar to how it works in `gaussian_high_pass`.